bake/bake.go

```diff
@@ -526,6 +527,12 @@ func updateContext(t *build.Inputs, inp *Input) {
 		t.ContextPath = inp.URL
 		return
 	}
+	if strings.HasPrefix(t.ContextPath, "cwd://") {
```

Comment:
Document `cwd` protocol

Reply:
I'm not sure if it is a good idea. I don't want users to write it in the bake file. If they do then they break the remote URL case. They should use the variable and this should be the internal behavior of the variable. We should be able to change that to `workdir://` for example and no old files should be broken.

We should document the builtin var, and also more docs for vars/attrs in general.

Reply:
> they break the remote URL

Ah yes right

> We should document the builtin var, and also more docs for vars/attrs in general.

:+1: